docs/deployment/linux/debian.txt

```diff
@@ -166,14 +117,7 @@ so:
 
     sh$ sudo systemctl COMMAND crate
 
-With Debian Wheezy (9.x), you must do this instead:
-
-.. code-block:: sh
-
-    sh$ sudo /etc/init.d/crate COMMAND
-
-In both instances, replace ``COMMAND`` with ``start``, ``stop``, ``restart``,
-``status``, etc.
+Replace ``COMMAND`` with ``start``, ``stop``, ``restart``, ``status``, etc.
```

Comment:
"Here, replace" ... "`status`, and so on."

(I know this wasn't in the original file, but we may as well improve it while we're here